src/index.js

```diff
@@ -31,7 +52,12 @@ const compileStylusFile = (jsFile, stylusFile) => {
     throw Error('Cannot find stylus file: ' + stylusFile);
   }
 
-  const stylusContent = fs.readFileSync(path, 'utf8');
+  let stylusContent = fs.readFileSync(path, 'utf8');
+
+  if (stylusContent.indexOf('~') !== -1) {
+    stylusContent = convertNodeModuleImport(stylusContent);
+  }
+
   return stylus(stylusContent)
     .include(dirname(path))
     .include(resolve('./node_modules'))
```

Comment:
since `./node_modules` is already in the include path, couldn't we just strip `~` and `~/` from the import string instead of having to do the whole `convertNodeModuleImport` function?